content/projects/kinodata-3D.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Kinodata-3D
+date: 2022-01-01
+publishdate: 2023-08-20
+weight: 10
+nav: false
+people:
+- key: michael.backenkoehler 
+- key: andrea.volkamer
+collaborators:
+- name: Joschka Groß
+  link: http://mosi.uni-saarland.de/people/joschka
+- name: Verena Wolf
+  link: http://mosi.uni-saarland.de/people/verena
+external_resources:
+- name: kinodata-3D
+  link: https://github.com/volkamerlab/kinodata-3D
+  icon: github
+- name: kinodata-3D-affinity-prediction
+  link: https://github.com/volkamerlab/kinodata-3D-affinity-prediction
+  icon: github
+funding:
+- name: NextAID
+  link: https://nextaid.cs.uni-saarland.de/
+---
+
+Structural machine learning models require large datasets for training. As such datasets are unavailable, we assess the use of in-silico structural docking data for machine learning. To this end, we perform template docking on a large kinase activity dataset (kinodata). We extensively compare affinity prediction models with access to the 3D complexes to various baseline models without access to this structure. We observe an increased performance of the model trained on the docked complexes.
```

Comment:
```suggestion
Machine learning - and especially deep learning - models require large datasets for training. As such datasets, especially those containing protein-ligand-complex information - are more rare in the drug design landscape, we assess the use of _in silico_ structural docking data for machine learning. 

To this end, we perform template docking using the OpenEye software on a large kinase activity dataset (kinodata) following the complex generation pipeline developed in kinoml. 

To asses the performance gain of using generated structural data, we extensively compare affinity prediction models with access to the 3D complexes to various baseline models without access to this structure. Overall, we observe an increased performance of the model trained on the docked complexes.
```

Reply:
can you add links to OpenEye, kinodata and kinoml?

Reply:
you could also add a sentence that links the generated data set back to Paula's project (now added in the mainline)